src/bento/components/bento-base-carousel/1.0/test/standalone/test-bento-base-carousel.js

```diff
@@ -0,0 +1,352 @@
+      expect(buttons).to.have.length(2);
+    });
+
+    it('should not render arrows when controls=never', async () => {
+      element.setAttribute('controls', 'never');
+      const userSuppliedChildren = setSlides(3);
+      userSuppliedChildren.forEach((child) => element.appendChild(child));
+      win.document.body.appendChild(element);
+
+      await element.getApi();
+
+      const renderedSlides = await getSlidesFromShadow();
+      expect(renderedSlides).to.have.ordered.members(userSuppliedChildren);
+      const buttons = element.shadowRoot.querySelectorAll('button');
+      expect(buttons).to.have.length(0);
+    });
+
+    it('should go to slide 0 when slide attr is mutated to 0', async () => {
+      const userSuppliedChildren = setSlides(3);
+      userSuppliedChildren.forEach((child) => element.appendChild(child));
+      win.document.body.appendChild(element);
+
+      await element.getApi();
+
+      await getSlidesFromShadow();
+
+      const scroller = element.shadowRoot.querySelector(
+        `[class*=${styles.scrollContainer}]`
+      );
+
+      element.setAttribute('slide', '1');
+      await waitFor(() => scroller.scrollLeft > 0, 'go to slide 1');
+
+      element.setAttribute('slide', '0');
+      await waitFor(() => scroller.scrollLeft == 0, 'returned to first slide');
+    });
+
+    it('should start at slide 1 with slide attr set to 1', async () => {
+      element.setAttribute('slide', '1');
+      const userSuppliedChildren = setSlides(3);
+      userSuppliedChildren.forEach((child) => element.appendChild(child));
+      win.document.body.appendChild(element);
+
+      await element.getApi();
+
+      await getSlidesFromShadow();
+
+      const scroller = element.shadowRoot.querySelector(
+        `[class*=${styles.scrollContainer}]`
+      );
+      await waitFor(() => scroller.scrollLeft > 0, 'render at slide 1');
+    });
+
+    // todo(kvchari): debug
+    // it('should respect outset-arrows even if controls=never', async () => {
+    //   element.setAttribute('controls', 'never');
+    //   element.setAttribute('outset-arrows', '');
+    //   const userSuppliedChildren = setSlides(3);
+    //   userSuppliedChildren.forEach((child) => element.appendChild(child));
+    //   win.document.body.appendChild(element);
+
+    //   await element.getApi();
+
+    //   const shadow = element.shadowRoot;
+
+    //   // Container is 300px and arrows each take up 50px after padding
+    //   expect(element.offsetWidth).to.equal(300);
+    //   const scroller = shadow.querySelector(
+    //     `[class*=${styles.hideScrollbar}]`
+    //   );
+    //   expect(scroller.offsetWidth).to.equal(200);
+
+    //   const prevButton = shadow.querySelector(`[class*=${styles.arrowPrev}]`);
+    //   expect(prevButton.offsetWidth).to.equal(36);
+    //   const nextButton = shadow.querySelector(`[class*=${styles.arrowNext}]`);
+    //   expect(nextButton.offsetWidth).to.equal(36);
+    // });
```

Comment:
Do you have a GitHub issue you can associate this with?
```suggestion
    it.skip('should respect outset-arrows even if controls=never', async () => {
      element.setAttribute('controls', 'never');
      element.setAttribute('outset-arrows', '');
      const userSuppliedChildren = setSlides(3);
      userSuppliedChildren.forEach((child) => element.appendChild(child));
      win.document.body.appendChild(element);

      await element.getApi();

      const shadow = element.shadowRoot;

      // Container is 300px and arrows each take up 50px after padding
      expect(element.offsetWidth).to.equal(300);
      const scroller = shadow.querySelector(
        `[class*=${styles.hideScrollbar}]`
      );
      expect(scroller.offsetWidth).to.equal(200);

      const prevButton = shadow.querySelector(`[class*=${styles.arrowPrev}]`);
      expect(prevButton.offsetWidth).to.equal(36);
      const nextButton = shadow.querySelector(`[class*=${styles.arrowNext}]`);
      expect(nextButton.offsetWidth).to.equal(36);
    });
```

Reply:
looks like it works when i just tested it now, so i'll re-add it